Cargo.lock

```diff
@@ -1,6 +1,6 @@
 [root]
 name = "cage"
-version = "0.2.0"
+version = "0.3.0"
```

Comment:
@dkastner The Rust convention for semver < 1.0 is to only bump the MINOR version for breaking changes. There's a bunch of machinery in `cage` itself based on this, to know when to refuse to load cage projects.

So this patch should 0.2.1. And we should also consider bringing in the open 3rd-party PR as well for this release; it's a nice bug fix and not too complicated last I looked.